### DISCUSSION TITLE
cannot add rtsp_simple_server

### DISCUSSION THREAD

**Discussion by kenneth27dunne** [fuatakgun/eufy_security#320] [General]
I keep getting an error saying "Invalid Add-on repository!" when I try to add https://github.com/fuatakgun/rtsp_simple_server can anyone help? I'm new to Home assistant and want to add my eufy doorbell to it but it seems to be more hassle than its worth at this stage

**Comment by fuatakgun** [comment]
This is definitely a hassle, but it is what it is.

I have no information on your setup and cannot see any logs, so won't be able to help.

But this has been working for many of us, i have just tried it now too, so it is definitely something local.
